src/views/Catalog.vue

```diff
@@ -73,6 +72,7 @@ export default defineComponent({
   data() {
     return {
       queryString: "",
+      count:0
```

Comment:
I think the count variable is not needed, as you are using vIndex for infinite scroll.